blogposts/2022/release-3.37.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Sourcegraph 3.37 release'
+publishDate: 2022-02-22T10:00-07:00
+description: 'Sourcegraph 3.37 introduces performance improvements for Code Intelligence, sharing for Notebooks, and a new UI for creating search contexts.'
+tags: [blog, release]
+slug: 'release/3.37'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.37/sourcegraph-3-37-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.37/sourcegraph-3-37-release.png
+changelogItems:
+  - description: 'Code in search results is now selectable (e.g. for copying). Just clicking on the code continues to open the corresponding file as it did before.'
```

Comment:
@iskyOS - is copying the only use case? If, so we can just say "Code in search results is now selectable for easier copying and reuse."

I also added "reuse" - does that feel too forced? 

"You can still click on the code to open the corresponding file."

Reply:
@abschu Yes, just for copying. I don't think resuse is too forced; that's the core use case here (copy+paste / reuse without having to go into the file).

```suggestion
  - description: 'Code in search results is now selectable for easier copying and reuse. You can still click on the code to open the corresponding file.'
```